### ISSUE TITLE
enable SSL termination

### ISSUE THREAD

**Issue by cppforlife** [shinji62/minio-dist-boshrelease#2]
may be colocated with nginx release? or does minio support its own SSL termination?

cc @dpb587

**Comment by shinji62** [comment]
Not sure to be honest I will investigates.

**Comment by dpb587** [comment]
Yes - if `certs/private.key` and `certs/public.crt` are present in minio's config directory TLS is used. Config dir can be configured with `-C {dir}`, and [related certs globals](https://github.com/minio/minio/blob/e5773e11c670a70264434c7837f30dd2106ad961/cmd/globals.go#L42-L46).
